### DISCUSSION TITLE
Humpback wiring

### DISCUSSION THREAD

**Discussion by SAYtan47** [FakeFishGames/Barotrauma#14514] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

There is an unused relay component in the humpback
![20240827121922_1](https://github.com/user-attachments/assets/39f40fcd-eabc-44f6-9b1d-e37eb86491f2)
It is placed in the floor right below the oxygen shelves

### Reproduction steps

1. Go in the submarine editor and choose humpback
2. Enable wiring mode and go to the diving suits
3. Here you go, the 4th relay component in a row isnt connected to anything

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/792d2ff339faaa0703e1b7e3e33d788b05461130 (commit in our private repo), fix coming up in the next update.
